File: java/TokoKomputer.java
public class TokoKomputer extends TokoElektronik {
    private String kodePC;
    private String spesifikasiPC;
    private String hargaPC;

    public TokoKomputer() {}

    // Setter & Getter
    public void setKodePC(String kodePC) {
        this.kodePC = kodePC;
    }
    public String getKodePC() {
        return kodePC;
    }

    public void setSpesifikasiPC(String spesifikasiPC) {
        this.spesifikasiPC = spesifikasiPC;
    }
    public String getSpesifikasiPC() {
        return spesifikasiPC;
    }

    public void setHargaPC(String hargaPC) {
        this.hargaPC = hargaPC;
    }
    public String getHargaPC() {
        return hargaPC;
    }

    // Add data komputer
    public void addDataKomputer(String namaToko, String alamatToko, String notelToko,
                                String kodePC, String spesifikasiPC, String hargaPC) {
        super.addData(namaToko, alamatToko, notelToko);
        this.kodePC = kodePC;
        this.spesifikasiPC = spesifikasiPC;
        this.hargaPC = hargaPC;
        System.out.println("Data Komputer berhasil ditambah!");
    }
}

File: python/TokoKomputer.py
from TokoElektronik import TokoElektronik

class TokoKomputer(TokoElektronik):
    def __init__(self, kode_pc="", spesifikasi_pc="", harga_pc="", **kwargs):
        super().__init__(**kwargs)
        self.kode_pc = kode_pc
        self.spesifikasi_pc = spesifikasi_pc
        self.harga_pc = harga_pc

    # Getter & Setter
    def set_kode_pc(self, kode_pc):
        self.kode_pc = kode_pc

    def get_kode_pc(self):
        return self.kode_pc

    def set_spesifikasi_pc(self, spesifikasi_pc):
        self.spesifikasi_pc = spesifikasi_pc

    def get_spesifikasi_pc(self):
        return self.spesifikasi_pc

    def set_harga_pc(self, harga_pc):
        self.harga_pc = harga_pc

    def get_harga_pc(self):
        return self.harga_pc

    # Add data
    def add_data_komputer(self, nama_toko, alamat_toko, notel_toko, kode_pc, spesifikasi_pc, harga_pc):
        super().add_data(nama_toko, alamat_toko, notel_toko)
        self.kode_pc = kode_pc
        self.spesifikasi_pc = spesifikasi_pc
        self.harga_pc = harga_pc
        print("Data Komputer berhasil ditambah!")
